### PR TITLE
Simplify code

### PR DESCRIPTION
**Pull request by forresst**
See [Node.js](https://github.com/nodejs/node/blob/master/lib/internal/readline/utils.js#L103..L137)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -2,48 +2,38 @@
 'use strict';
 
 const isFullwidthCodePoint = codePoint => {
-	if (Number.isNaN(codePoint)) {
```
Comment (sindresorhus) [inline index.js:5]: I would prefer to have:

```js
if (!Number.isInteger(codePoint)) {
	return false;
}
```

here.

**Comment by forresst** [comment]
@sindresorhus Does this correction seems better to you?

File: index.js (hunk 1 above)
Comment (forresst) [reply index.js:5]: I did what you asked me to do, but for my personal knowledge, what is the advantage of separating the test: `!Number.isInteger(codePoint)`

Comment (sindresorhus) [reply index.js:5]: Just my preference, and it's slightly more readable. I prefer having type "assertions" separated from the actual logic.

**Comment by sindresorhus** [comment]
Yup, thanks 👌

**Pull request merged**